### ISSUE TITLE
If a player leaves with zombie blood, it breaks all health regeneration

### ISSUE THREAD

**Issue by globalcringe** [Zet0rz/nzombies#301]
Had this happen recently.
If a player leaves the server while they have zombie blood, it seems that everything under sh_ent_ext.lua under gamemodes/nzombies/gamemode/enemies/ completely breaks, stopping things like health-regeneration.
Here's the error. It appears to happen every tick.

```
[ERROR] addons/nzombies-master-workshop/gamemodes/nzombies/gamemode/enemies/sh_ent_ext.lua:20: Tried to use a NULL entity!
1. __newindex - [C]:-1
2. SetTargetPriority - addons/nzombies-master-workshop/gamemodes/nzombies/gamemode/enemies/sh_ent_ext.lua:20
3. func - addons/nzombies-master-workshop/gamemodes/nzombies/gamemode/powerups/sh_powerups.lua:245
4. fn - addons/nzombies-master-workshop/gamemodes/nzombies/gamemode/powerups/sv_powerups.lua:16
```

I do apologize if I don't reply immediately, as I rarely check my e-mail.

**Comment by Zet0rz** [comment]
Oh wow, thanks for pointing that out! The powerup and any other timer-based system really needs to check if their owner is valid first.

**Comment by globalcringe** [comment]
Hey, I don't know how I could've sent you this, so I'm throwing it here.
I ripped higher-quality versions of some songs currently used in nZombies! Specifically, dog rounds starting and ending. I also ripped some of the perk machine jingles, as well as three game over songs from Call of Duty: World at War.
Here's the rips if you want to integrate them into your content addon. I've already named a few.
https://dl.dropboxusercontent.com/s/t6jt47kdkhui6i0/rips.zip
